components/os/unix.go

```diff
@@ -40,11 +44,29 @@ func getDefaultInstanceType(env config.Environment, arch Architecture) string {
 	case ARM64Arch:
 		return env.DefaultARMInstanceType()
 	default:
-		panic("Architecture not supportede")
+		panic("Architecture not supported")
 	}
 }
 
 func getUnixInstallFormatString(scriptName string, version AgentVersion) string {
+	if version.PipelineID != "" {
+		testEnvVars := []string{}
+		testEnvVars = append(testEnvVars, "TESTING_APT_URL=apttesting.datad0g.com")
```

Comment:
Note: You can reduce the code size by initialising the array directly
```
testEnvVars := []string{
     "TESTING_APT_URL=apttesting.datad0g.com",
    "fmt.Sprintf(`TESTING_APT_REPO_VERSION="%v-a7 7"`, version.PipelineID)",
.....
}

Reply:
I do like the way it is, because it makes the docs more clear for meaning of each line, so I may reduce a bit, but in my opinion it will look less clear what is going on